packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance/ShortAssistanceLinkItem.tsx

```diff
@@ -25,22 +27,28 @@ export const ShortAssistanceLinkItem: FunctionComponent<Props> = ({ node, isSele
     } else if (node.url) {
         url = node.url as string;
     }
-
+    
 
     return (
-        <li className={`flex p-1 justify-center ${isSelected ? style.sidebar_menu_items_selected : ''} ${style.sidebar_menu_items}`} role="menuitem" >
-            <a
-                onClick={node.onClick}
-                href={url}
-                target={node.isExternal ? '_blank' : '_top'}
-                rel={node.isExternal ? 'noopener noreferrer' : ''}
-                title={t(node.translation)}
-                id={node.id}
-                role="link"
-                className='d-flex items-center justify-center'
-            >
-                <SvgIconWrapper name={node.svgIcon} height={24} width={24} className='fill-white block' />
-            </a>
-        </li>
+        <OsdsButton
+            onClick={node.onClick}
+            href={url}
+            contrasted
+            color={ODS_THEME_COLOR_INTENT.primary}
+            target={node.isExternal ? OdsHTMLAnchorElementTarget._blank : OdsHTMLAnchorElementTarget._top}
+            rel={node.isExternal ? OdsHTMLAnchorElementRel.noopener : undefined}
+            id={node.id}
+            role="link"
+            size={ODS_BUTTON_SIZE.sm}
+            textAlign={ODS_BUTTON_TEXT_ALIGN.start}
+        >
+            {t(node.translation)}
+            {node.isExternal && (
+                <span
+                    aria-hidden="true"
+                    className={`${style.sidebar_external} oui-icon oui-icon-external-link`}
+                ></span>
+            )}
+        </OsdsButton>
```

Comment:
As recommended by ODS (https://ovh.github.io/design-system/latest/iframe.html?viewMode=docs&id=ods-components-button--documentation#dos--donts), we should not use button as links,